api/shared/nodenetworkconfigurationenactment_types.go

```diff
@@ -13,10 +14,35 @@ type NodeNetworkConfigurationEnactmentStatus struct {
 	// the policy desiredState as template
 	DesiredState State `json:"desiredState,omitempty"`
 
+	// CapturedState The captured state from NMPolicy
+	// after processing the capture at NNCP
+	NMPolicyStatus NodeNetworkConfigurationEnactmentNMPolicyStatus `json:"nmpolicyStatus,omitempty"`
```

Comment:
I have mixed feeling about the name. We don't mention nmstate in our API, I think we should not mention nmpolicy neither. It also makes it confusing as there is nmpolicy as a part of policy. Could we grant it a more generic name? "templating" or something similar.

Calling in the namemaster @EdDev

Reply:
Maybe `CaptureStatus`?

Reply:
Capture sounds good to me, thanks o/